core/src/include/ModelArrayRef.old.hpp

```diff
@@ -0,0 +1,265 @@
+/*!
```

Comment:
did you want to keep this file in the repository - or is it an accidental commit?

Reply:
Of course I didn't want to keep it